etc/adminhtml/system/payment_methods/local_payment_methods/advanced_options.xml

```diff
@@ -26,32 +26,42 @@
+        <label>Custom shopper country</label>
+        <depends>
+            <field id="shopper_country">1</field>
+        </depends>
+        <source_model>Magento\Directory\Model\Config\Source\Country</source_model>
+        <validate>required-entry</validate>
+        <config_path>payment/adyen_hpp/country_code</config_path>
+    </field>
+    <field id="ratepay_id" translate="label" type="text" sortOrder="30" showInDefault="1" showInWebsite="1" showInStore="1">
         <label>RatePAY Device Ident SId</label>
         <tooltip>Unique RatePAY Id provided by RatePAY integration consultant</tooltip>
         <config_path>payment/adyen_hpp/ratepay_id</config_path>
     </field>
-    <field id="split_payments_refund_strategy" translate="label" type="select" sortOrder="30" showInDefault="1" showInWebsite="1" showInStore="1">
+
+    <field id="split_payments_refund_strategy" translate="label" type="select" sortOrder="40" showInDefault="1" showInWebsite="1" showInStore="1">
         <label>Refunds that include gift cards</label>
         <comment><![CDATA[Learn more about refunds that include gift cards on <a href="https://docs.adyen.com/plugins/magento-2/set-up-the-plugin-in-magento/#split-payment" target="_blank">Adyen documentation</a>.]]></comment>
         <source_model>Adyen\Payment\Model\Config\Source\SplitPaymentRefundStrategy</source_model>
         <config_path>payment/adyen_abstract/split_payments_refund_strategy</config_path>
     </field>
-    <field id="allowspecific_local_payment_methods" translate="label" type="allowspecific" sortOrder="40" showInDefault="1" showInWebsite="1" showInStore="0">
+    <field id="allowspecific_local_payment_methods" translate="label" type="allowspecific" sortOrder="50" showInDefault="1" showInWebsite="1" showInStore="0">
         <label>Applicable countries</label>
         <source_model>Magento\Payment\Model\Config\Source\Allspecificcountries</source_model>
         <config_path>payment/adyen_hpp/allowspecific</config_path>
     </field>
-    <field id="specificcountry" translate="label" type="multiselect" sortOrder="50" showInDefault="1" showInWebsite="1" showInStore="0">
-        <label>Payment from specific countries</label>
-        <depends>
-            <field id="allowspecific_local_payment_methods">1</field>
-        </depends>
-        <source_model>Magento\Directory\Model\Config\Source\Country</source_model>
-        <can_be_empty>1</can_be_empty>
-        <config_path>payment/adyen_hpp/specificcountry</config_path>
+    <field id="specificcountry" translate="label" type="multiselect" sortOrder="60" showInDefault="1" showInWebsite="1" showInStore="0">
+    <label>Payment from specific countries</label>
+    <depends>
+        <field id="allowspecific_local_payment_methods">1</field>
+    </depends>
+    <source_model>Magento\Directory\Model\Config\Source\Country</source_model>
+    <can_be_empty>1</can_be_empty>
```

Comment:
If the country_code dropdown shows we can validate that it has a selection with this line
```suggestion
    <validate>required-entry</validate>
```